File: The Minion Game/The_Minion_game.py
def minion_game( orignal_word ):
    length_str = len(orignal_word)

    S=0
    K=0

    for i, x in enumerate(orignal_word):
        
        if x in ["A","a","E","e","I","i","O","o","U","u"]: 
            K+= (length_str-i)
        else:
            S+= (length_str-i)


    if(S > K):
        print("Stuart", S)
    elif(S < K):
        print("Kevin", K)
    else:
        print("Draw")


## Done

File: The Minion Game/The_Minion_game.cpp
#include<iostream>

using namespace std;

bool is_vowel( char s ){

    char vovel[] = {'a', 'e', 'i', 'o', 'u'};
    int flag = 0;
    for(int i = 0; i<5; i++){
        if( s == vovel[i]){   
            flag = 1;
            break;
        }
    }

    if( flag == 0 )
        return false;
    else
        return true;

}

int main(){
    
    string word = "Banana";

    int length = word.length();

    // Score for Stuart (consonent)
    long stuart = 0;
    for(int i=0; i<length; i++)
    {
        if(!is_vowel(word[i])){
            //cout<<length<<"-"<<i<<" ="<<length-i<<endl;
            stuart+=(length-i);
        }
    }

    // Score for Stuart (consonent)
    long kevin = 0;
    for(int i=0; i<length; i++)
    {
        if(is_vowel(word[i])){
            //cout<<length<<"-"<<i<<" ="<<length-i<<endl;
            kevin+=(length-i);
        }
    }

    if(stuart>kevin){
        cout<<"Stuart "<<stuart;
    }
    else if (kevin>stuart){
        cout<<"Kevin "<<kevin;
    }
    else{
        cout<<"Draw";
    }

    return 0;
}


//Done